src/main/java/frontend/AdvancedTab.java

```diff
@@ -0,0 +1,35 @@
+package frontend;
+
+import javax.swing.*;
+import java.awt.*;
+
+class AdvancedTab extends JComponent {
```

Comment:
I see that we've abstracted tabbed panes to new class files, but there still isn't anything that allows us to manipulate elements within them after instantiation.

This isn't necessarily an immediate problem, but when do we plan on fixing it?